File: Java/poo/drivingRecord/library/licenseC.java
package poo.drivingRecord.library;

    public class licenseC{
        private String typeLicense;
        private int cost;
        private int daysDuration;

        public licenseC(String typeLicense, int cost, int daysDuration){
            this.typeLicense = typeLicense;
            this.cost = cost;
            this.daysDuration = daysDuration;
        }
        public String getTypeLicense(){
            return this.typeLicense;
        }
        public int getCost(){
            return this.cost;
        }
        public int getDaysDuration(){
            return this.daysDuration;
        }
    }

File: Php/poo/drivingRecord/library/licenseC.php
<?php

class licenseC extends student{
    private string $typeLicense;
    private int $cost;
    private int $daysDuration;

    public function __construct(string $typeLicense, int $cost, int $daysDuration){
        $this->typeLicense = $typeLicense;
        $this->cost = $cost;
        $this->daysDuration = $daysDuration;
    }
    public function getTypeLicense(): string{
        return $this->typeLicense;
    }
    public function getCost(): int{
        return $this->cost;
    }
    public function getDaysDuration(): int{
        return $this->daysDuration;
    }
}
